src/descriptors/power_system_structs.json

```
@@ -94,36 +94,30 @@
       "supertype": "OperationalCost"
     },
     {
-      "struct_name": "MultiStartCost",
-      "docstring": "Data Structure Operational Cost Data which includes fixed, variable cost, multiple start up cost and stop costs.",
+      "struct_name": "MarketBidCost",
+      "docstring": "Data Structure Operational Cost to reflect market bids of energy and ancilliary services.\nCompatible with most US Market bidding mechanisms",
       "fields": [
         {
-          "name": "variable",
+          "name": "energy_bid",
```

```suggestion
          "name": "variable",
```
I would also not change the name here as effectively it's the same as `variable` in `ThreePartCost` or other types. Better to be consistent and have documentation to make it clear if we judge necessary.